server/controllers/list.js

```diff
@@ -456,16 +494,232 @@ const removeFromFavourites = (req, resp) => {
   );
 };
 
+const removeOwner = (req, resp) => {
+  const { id: listId } = req.params;
+  const { userId } = req.body;
+  const {
+    user: { _id: ownerId }
+  } = req;
+
+  List.findOneAndUpdate(
+    { _id: listId, ownerIds: { $all: [ownerId, userId] } },
+    { $pull: { ownerIds: userId } },
+    (err, doc) => {
+      if (err) {
+        return resp.status(400).send({
+          message: "Can't remove owner from list."
+        });
+      }
+
+      if (doc) {
+        return resp.status(200).send({
+          message: 'Owner successfully removed from list.'
+        });
+      }
+
+      resp.status(404).send({ message: 'List data not found.' });
+    }
+  );
+};
+
+const removeMember = (req, resp) => {
+  const { id: listId } = req.params;
+  const { userId } = req.body;
+  const {
+    user: { _id: ownerId }
+  } = req;
+
+  List.findOneAndUpdate(
+    { _id: listId, ownerIds: ownerId, memberIds: userId },
+    { $pull: { memberIds: userId } },
+    (err, doc) => {
+      if (err) {
+        return resp.status(400).send({
+          message: "Can't remove member from list."
+        });
+      }
+
+      if (doc) {
+        return resp.status(200).send({
+          message: 'Member successfully removed from list.'
+        });
+      }
+
+      resp.status(404).send({ message: 'List data not found.' });
+    }
+  );
+};
+
+const changeToOwner = (req, resp) => {
+  const { id: listId } = req.params;
+  const { userId } = req.body;
+
+  const {
+    user: { _id: ownerId }
+  } = req;
+
+  List.findOne({ _id: listId, ownerIds: ownerId })
+    .populate('cohortId', 'memberIds ownerIds')
+    .exec()
+    .then(doc => {
+      if (!doc) {
+        throw new BadRequestException("Can't set user as a list's owner.");
+      }
+
+      const { cohortId: cohort, isPrivate, memberIds } = doc;
+      const isCohortMember = checkIfCohortMember(cohort, userId);
```

Comment:
you can make it as :
`isNotCohortMember = !checkIfCohortMember(cohort, userId);`